### PR TITLE
Kogito 2549 - add test on container

### PR DESCRIPTION
**Pull request by r00ta**
Jira: https://issues.redhat.com/browse/KOGITO-2549

This PR adds a test on the kogito application that is run on a container. 

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by r00ta** [comment]
jenkins retest this please

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java [hunk 1]
```diff
@@ -65,4 +66,23 @@ public void testGrafanaDashboards() {
...
     }
+
+    @Test
+    public void testKogitoContainerIsDeployedAndResponsive() {
+        String body = "{\n" +
+                "\"Client\": {\"age\": 43,\"salary\": 1950, \"existing payments\": 100},\n" +
+                "\"Loan\": {\"duration\": 15,\"installment\": 180}, \n" +
+                "\"SupremeDirector\" : \"Yes\", \n" +
+                "\"Bribe\": 1000\n" +
+                "}";
```
Comment (danielezonca) [inline dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java]: Can you please remove all `\n`? We don't need to pretty print the Json :)

**Review by danielezonca — APPROVED** [review]
Approving with minor comment :)

**Review by jiripetrlik — APPROVED** [review]
@r00ta 
Looks good to me! Thank you for extending these tests.

**Pull request merged**